lib/cli/commands/subscribeorders.ts

```diff
@@ -15,20 +15,56 @@ export const builder = {
 };
 
 export const handler = (argv: Arguments) => {
+  ensureConnection(argv, true);
+};
+
+const ensureConnection = (argv: Arguments, printError?: boolean) => {
+  loadXudClient(argv).waitForReady(Number.POSITIVE_INFINITY, (error: Error | null) => {
```

Comment:
Do we need to reload the XudClient every time? Or can we just load it once, maintain the reference to it,  and call `waitForReady` as needed.

Reply:
good catch